subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -515,6 +536,9 @@ uint8_t ll_adv_params_set(uint16_t interval, uint8_t adv_type,
 		/* No CTEInfo field in primary channel PDU */
 
 		/* TargetA */
+		if (pri_hdr_prev.tgt_addr) {
+			pri_dptr_prev -= BDADDR_SIZE;
```

Comment:
See previous comment. Seems to also apply other places for tgt_addr.

Reply:
TARGETA_SIZE is only used in the context of calculating legacy PDU size, and all other instances use BDADDR_SIZE.